Product/Production/Gateway/DocumentDataSubmission_10/src/main/resources/documentdatasubmission10.properties

```diff
@@ -1,2 +1,2 @@
-docdatasubmission.inboundDocDataSubmission=gov.hhs.fha.nhinc.docdatasubmission.inbound.StandardInboundDocDataSubmission
-docdatasubmission.outboundDocDataSubmission=gov.hhs.fha.nhinc.docdatasubmission.outbound.StandardOutboundDocDataSubmission
\ No newline at end of file
+docdatasubmission.inboundDocDataSubmission=gov.hhs.fha.nhinc.docdatasubmission.inbound.PassthroughInboundDocDataSubmission
+docdatasubmission.outboundDocDataSubmission=gov.hhs.fha.nhinc.docdatasubmission.outbound.PassthroughInboundDocDataSubmission
```

Comment:
@TabassumJafri this slipped by me. Slack is down, so using this as an alternative. Outbound is mapping to Inbound here in the properties file.